spng/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2018-2023, Randy <randy408@protonmail.com>
```

Comment:
Could you add this to the copyright section in README.md?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1708/commits/66de5149776a4210e03a31dc5e1f95d24ee65df9